modules/game_forge/game_forge.css

```
@@ -372,9 +397,13 @@
 }
 
 .history-footer {
-  height: 20px;
-  width: 465px;
+
   display: flex;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)